### PR TITLE
🌐 Add French translation for `docs/fr/docs/benchmarks.md`

### PR DESCRIPTION
**Pull request by clemsau**
Hello 👋,

This is a PR to translate `docs/benchmarks.md` to French, as told in the issue #1972.

Sorry for the typo in the commit message.

See the formatted document [here](https://github.com/ClemSau/fastapi/blob/master/docs/fr/docs/benchmarks.md)

Thanks for those who take the time to validate it.

### CODE REVIEW

**Comment by Jefidev** [comment]
Hi, I reviewed your text and for me it is ok. I just have a suggestion :
I think that it is prettier to change this sentences :
[...] les applications FastAPI tournant sous Uvicorn comme **un** des frameworks Python les plus rapides disponibles,

to this :
[...] les applications FastAPI tournant sous Uvicorn comme **étant l'un** des frameworks Python les plus rapides disponibles,

**Comment by Jefidev** [comment]
Oh I didn't notice at first glance but you forgot to update the `mkdocs.yml`.

**Comment by ClemSau** [comment]
I have taken into account your proposal @Jefidev , and I updated the `mkdocs.yml` .

File: docs/fr/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* **Uvicorn**: un serveur ASGI
+    * **Starlette**: (utilise Uvicorn) un micro-framework web
+        * **FastAPI**: (uses Starlette) un micro-framework pour API disposant de fonctionnalités additionnels pour la création d'API, avec la validation des données, etc.
+
+* **Uvicorn**:
+    * A les meilleures performances, étant donné qu'il n'a pas beaucoup de code mis-à-part le serveur en lui-même.
+    * On n'écrit pas une application avec uniquement Uvicorn. Cela signifie que le code devrait inclure plus ou moins, au minimum, tout le code offert par Starlette (ou **FastAPI**). Et si on fais cela, l'application finale apportera les mêmes complications que si on avait utilisées un framework et que l'on avait minimisé la quantité de code et de bugs.
+    * Si on compare Uvicorn, il faut le comparer à d'autre applications de serveurs comme Daphne, Hypercorn, uWSGI, etc.
+* **Starlette**:
+    * A les seconde meilleures performances après Uvicorn. Starlette utilise en réalité Uvicorn. De ce fait, il ne peut qu’être plus "lent" qu'Uvicorn en requièrent l'exécution de plus de code.
```
Comment (rjNemo) [inline docs/fr/docs/benchmarks.md:26]:
```suggestion
    * A les seconde meilleures performances après Uvicorn. Starlette utilise en réalité Uvicorn. De ce fait, il ne peut qu’être plus "lent" qu'Uvicorn car il requiert l'exécution de plus de code.
```

**Comment by rjNemo** [comment]
I left a few comments. To me it would improve the translation. However I think this PR is good as it is.

File: docs/fr/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    * Cependant il nous apporte les outils pour construire une application web simple, avec un routage basé sur des chemins, etc.
+    * Si on compare Starlette, il faut le comparer à d'autres frameworks web (ou micorframework) comme Sanic, Flask, Django, etc.
+* **FastAPI**:
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:29]:
```suggestion
* **FastAPI** :
```

File: docs/fr/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+* **Uvicorn**: un serveur ASGI
+    * **Starlette**: (utilise Uvicorn) un micro-framework web
+        * **FastAPI**: (uses Starlette) un micro-framework pour API disposant de fonctionnalités additionnels pour la création d'API, avec la validation des données, etc.
+
+* **Uvicorn**:
+    * A les meilleures performances, étant donné qu'il n'a pas beaucoup de code mis-à-part le serveur en lui-même.
+    * On n'écrit pas une application avec uniquement Uvicorn. Cela signifie que le code devrait inclure plus ou moins, au minimum, tout le code offert par Starlette (ou **FastAPI**). Et si on fais cela, l'application finale apportera les mêmes complications que si on avait utilisées un framework et que l'on avait minimisé la quantité de code et de bugs.
+    * Si on compare Uvicorn, il faut le comparer à d'autre applications de serveurs comme Daphne, Hypercorn, uWSGI, etc.
+* **Starlette**:
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:25]:
```suggestion
* **Starlette** :
```

File: docs/fr/docs/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Plus le problème résolu par un outil est simple, mieux seront les performances obtenues. Et la plupart des tests de performance ne prennent pas en compte les fonctionnalités additionnelles fournies par les outils.
+
+La hiérarchie est la suivante :
+
+* **Uvicorn**: un serveur ASGI
+    * **Starlette**: (utilise Uvicorn) un micro-framework web
+        * **FastAPI**: (uses Starlette) un micro-framework pour API disposant de fonctionnalités additionnels pour la création d'API, avec la validation des données, etc.
+
+* **Uvicorn**:
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:21]:
```suggestion
* **Uvicorn** :
```

File: docs/fr/docs/benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Lorsque vous vérifiez les tests de performance, il est commun de voir plusieurs outils de différents types comparés comme équivalents.
+
+En particulier, on voit Uvicorn, Starlette et FastAPI comparés (parmi de nombreux autres outils).
+
+Plus le problème résolu par un outil est simple, mieux seront les performances obtenues. Et la plupart des tests de performance ne prennent pas en compte les fonctionnalités additionnelles fournies par les outils.
+
+La hiérarchie est la suivante :
+
+* **Uvicorn**: un serveur ASGI
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:17]:
```suggestion
* **Uvicorn** : un serveur ASGI
```

File: docs/fr/docs/benchmarks.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    * **Starlette**: (utilise Uvicorn) un micro-framework web
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:18]:
```suggestion
    * **Starlette** : (utilise Uvicorn) un micro-framework web
```

File: docs/fr/docs/benchmarks.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        * **FastAPI**: (uses Starlette) un micro-framework pour API disposant de fonctionnalités additionnels pour la création d'API, avec la validation des données, etc.
```
[the remark this answers is not in the document]
Comment (Smlep) [reply docs/fr/docs/benchmarks.md:19]: Also add a space before the `:`.

File: docs/fr/docs/benchmarks.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    * FastAPI apporte des fonctionnalités supplémentaires à Starlette. Des fonctionnalités qui sont nécessaires presque systématiquement lors de la création d'une API, comme la validation des données, la sérialisation. En utilisant FastAPI, on obtient une documentation automatiquement (qui ne requiert aucune manipulation pour être mise en place).
+    * Si on n'utilisait pas FastAPI mais directement Starlette (ou un outil équivalent comme Sanic, Flask, Responder, etc) il faudrait implémenter la validation des données et la sérialisation par nous-même. Le résultat serait donc le même dans les deux cas mais du travail supplémentaire serait à réaliser avec Starlette, surtout en considérant que la validation des données et la sérialisation représentent la plus grande quantité de code à écrire dans une application.
+    * De ce fait, en utilisant FastAPI on minimise le temps de développement, les bugs, le nombre de lignes de code, et on obtient les mêmes performances (si ce n'est de meilleurs performances) que l'on aurait pu avoir sans ce framework (en ayant à implémenter de nombreuses fonctionnalités importantes par nous même).
```
Comment (Smlep) [inline docs/fr/docs/benchmarks.md:33]:
```suggestion
    * De ce fait, en utilisant FastAPI on minimise le temps de développement, les bugs, le nombre de lignes de code, et on obtient les mêmes performances (si ce n'est de meilleurs performances) que l'on aurait pu avoir sans ce framework (en ayant à implémenter de nombreuses fonctionnalités importantes par nous-mêmes).
```

Comment (Smlep) [inline docs/fr/docs/benchmarks.md:33]:
```suggestion
    * De ce fait, en utilisant FastAPI on minimise le temps de développement, les bugs, le nombre de lignes de code, et on obtient les mêmes performances (si ce n'est de meilleurs performances) que l'on aurait pu avoir sans ce framework (en ayant à implémenter de nombreuses fonctionnalités importantes par nous-mêmes).
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7e55a1f090c632756c859a5a9d489ac6471c5ba4 at: https://6499cda889d3ab086b01d304--fastapi.netlify.app

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1d30a7ef3e8b1321f86652551e4c834f210bfb91 at: https://64b4ffeeb4ec054a27d6c0d8--fastapi.netlify.app

**Review by emmrichard — APPROVED** [review]

**Review by rjNemo — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4da8b7ce54b7b82197a638451f21dfdf686602d1 at: https://64b652e4723d8f44d0b9ed18--fastapi.netlify.app

**Comment by tiangolo** [comment]
Thank you @clemsau! 🍰 

And thanks for the reviews @Smlep, @emmrichard, @axel584, @rjNemo ☕

**Pull request merged**